x.toml

```diff
@@ -12,12 +12,14 @@ flags = "-Zfeatures=all"
 bucket = "ci-artifacts.libra.org"
 prefix = "sccache/libra/"
 public = true
+region = "us-west-2"
+endpoint = "https://s3-us-west-2.amazonaws.com"
```

Comment:
needed when not using authentication as normally aws would redirect an unauthenticated read of https://ci-artifacts.libra.org/ to https://ci-artifacts.libra.org.s3-us-west-2.amazonaws.com/ which doesn't have a valid cert.  So instead forcing the use of https://s3-us-west-2.amazonaws.com/ci-artifacts.libra.org/ as the bucket location.